java/core/page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/support/DesiredCapabilitiesParser.java

```diff
@@ -77,14 +74,19 @@ public DesiredCapabilities parse() {
                         }
                         break;
                     default:
-                        options.put(optionsCapability, Props.get(rawCapabilityKey));
+                        options.put(optionsCapability, capabilityValue);
                         break;
                 }
                 if (!options.isEmpty()) {
                     capabilities.setCapability(ChromeOptions.CAPABILITY, options);
                 }
             } else {
-                capabilities.setCapability(capability, Props.get(rawCapabilityKey));
+                if ("true".equals(capabilityValue.toLowerCase()) ||
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)